packages/web-components/src/components/ic-side-navigation/ic-side-navigation.css

```diff
@@ -605,3 +605,9 @@
     box-shadow: none;
   }
 }
+
+@media (forced-colors: active) {
+  .side-navigation {
+    border-right: var(--ic-hc-border);
+  }
+}
```

Comment:
Do we think we need a border on the right of the application name section at the top as well?
![image](https://user-images.githubusercontent.com/111882109/205277558-e59ff49a-96b8-4172-8134-04654c173da8.png)